From: Valentin Vidic <vvidic@debian.org>
Date: Mon, 19 Oct 2020 22:01:38 +0200
Subject: Fix gcc build warnings

---
 fence/stonith_helper.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/fence/stonith_helper.c b/fence/stonith_helper.c
index 3a0768a..8bdabaa 100644
--- a/fence/stonith_helper.c
+++ b/fence/stonith_helper.c
@@ -8,6 +8,7 @@
 
 #include <stdio.h>
 #include <stdint.h>
+#include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
 #include <syslog.h>
@@ -24,7 +25,7 @@ static int get_options(int argc, char *argv[])
 	char arg[MAX_ARG_LEN];
 	char key[MAX_ARG_LEN];
 	char val[MAX_ARG_LEN];
-	char c;
+	int c;
 	int rv;
 
 	if (argc > 1) {
